pkg/bootstrap/types.go

```diff
@@ -54,17 +54,22 @@ type OVNConfigBoostrapResult struct {
 // OVNUpdateStatus contains the status of existing daemonset
 // or statefulset that are maily used by upgrade process
 type OVNUpdateStatus struct {
-	Kind         string
-	Namespace    string
-	Name         string
-	Version      string
-	IPFamilyMode string
-	Progressing  bool
+	Kind                 string
+	Namespace            string
+	Name                 string
+	Version              string
+	IPFamilyMode         string
+	ClusterNetworkCIDRs  string
+	Progressing          bool
+	InterConnectEnabled  bool
+	InterConnectZoneMode string
```

Comment:
nit: add comments (at least for zoneMode what that means and what are the possible values)